x/warp/predicate_test.go

```diff
@@ -348,10 +347,10 @@ func TestInvalidWarpMessage(t *testing.T) {
 				PChainHeight: 1,
 			},
 		},
-		StorageSlots: [][]byte{predicateBytes},
-		Gas:          GasCostPerSignatureVerification + uint64(len(predicateBytes))*GasCostPerWarpMessageBytes + uint64(numKeys)*GasCostPerWarpSigner,
-		GasErr:       errInvalidWarpMsg,
-		PredicateRes: set.NewBits(0).Bytes(), // Won't be reached
+		PredicateBytes: predicateBytes,
+		Gas:            GasCostPerSignatureVerification + uint64(len(predicateBytes))*GasCostPerWarpMessageBytes + uint64(numKeys)*GasCostPerWarpSigner,
+		GasErr:         errInvalidWarpMsg,
+		ExpectedRes:    true, // Won't be reached
```

Comment:
Should this be updated to false even though it won't be reached to match the previous value of including 0 in the bitset for this test?

Reply:
Could also remove this line altogether since it won't be reached (which would also set it to false)

Reply:
removed